src/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixer.php

```diff
@@ -91,7 +91,13 @@ public function configure(array $configuration = null)
                     $this->tokenKindCallbackMap[CT_USE_TRAIT] = 'removeBetweenUse';
                     break;
                 case 'curly_brace_open':
-                    $this->tokenEqualsMap['{'] = 'fixAfterToken';
+                    $this->tokenEqualsMap['{'] = 'fixAfterTokenIfMultiLineBody'; // i.e. not: CT_ARRAY_INDEX_CURLY_BRACE_OPEN
+                    break;
+                case 'square_brace_open':
+                    $this->tokenKindCallbackMap[CT_ARRAY_SQUARE_BRACE_OPEN] = 'fixAfterTokenIfMultiLineBody'; // typeless '[' tokens should not be fixed (to rare)
```

Comment:
I think something like:
```php
$a[

1,
2,
3,
];
```

happens (I'm looking @ Laravel configuration stuff).
But something like;
```php
$a = $b[

1];
```

is rare and would get fixed to;

```php
$a = $b[
1];
```

which is not much better ;)